herd/action.mli

```diff
@@ -93,7 +93,7 @@ module type S = sig
 
 (* Commits *)
   val is_bcc : action -> bool
-  val is_pred : action -> bool
+  val is_pred : ?_m:string option -> action -> bool
```

Comment:
I think I had to use underscore, otherwise I get warnings, as some implementations don't use it. But we can certainly use something more elaborate than `_m`..

Reply:
To avoid such warnings, you may use the `ignore` construct, typically:
```OCaml
let f x = ignore x; 0
```
Yields no warning, like
```OCaml
let f _x =  0
```

BTW, a more direct coding of  `is_pred` is possible:
```OCaml
let is_pred ?text = function
    | Commit (Pred, m0) ->
       Option.is_none text
       || Option.equal String.equal text m0
  | _ -> false
```
Type would then be  `val is_pred : ?text:string -> action -> bool`.
(See the `bump` example, at the end of  [this section of the OCaml manual](https://v2.ocaml.org/releases/5.1/htmlman/lablexamples.html#s%3Aoptional-arguments))